test/coretasks/test_coretasks_sasl.py

```diff
@@ -253,6 +253,26 @@ def test_sasl_plain_no_password(
     ), 'No password is a configuration error and the bot must quit.'
 
 
+# https://github.com/sopel-irc/sopel/issues/2560
```

Comment:
Nit: I don't think this is necessary, otherwise we would have hundreds of comments like this in the tests.

```suggestion
```

If you want to keep it, I'd rather see it in the test's docstring.